### ISSUE TITLE
Unhandled Exception (YSOD) logging in via OpenID

### ISSUE THREAD

**Issue by factormystic** [JabbR/JabbR#395]
_(Thank you for supporting OpenID!)_

Guessing it might be the whitespace in my username, but the system should be resilient to this input without a ysod

```
Exception Details: System.InvalidOperationException: 'Factor Mystic' is not a valid user name.

Source Error: 

An unhandled exception was generated during the execution of the current web request. Information regarding the origin and location of the exception can be identified using the exception stack trace below.

Stack Trace: 


[InvalidOperationException: 'Factor Mystic' is not a valid user name.]
   JabbR.Services.ChatService.AddUser(String userName, String identity, String email) in C:\dev\github\JabbR\JabbR\Services\ChatService.cs:287
   JabbR.Auth.Login.ProcessRequest(HttpContext context) in C:\dev\github\JabbR\JabbR\Auth\Login.ashx.cs:93
   System.Web.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute() +625
   System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously) +270
```

**Comment by davidfowl** [comment]
Ah good one. We maybe need to fix up your name.

**Comment by samandmoore** [comment]
this was fixed in PR #432 :)
